oauth2-server/src/test/java/com/clouway/oauth2/IssueNewTokenForClientTest.java

```diff
@@ -94,20 +94,13 @@ public void tokenCannotBeIssued() throws Exception {
 
       oneOf(identityFinder).findIdentity(with(any(String.class)), with(any(GrantType.class)), with(any(DateTime.class)));
       will(returnValue(Optional.of(identity)));
-
```

Comment:
It indicates a smell in the code as this check need to be performed here which actually need to be considered.